up42/storage.py

```diff
@@ -204,28 +205,46 @@ def get_assets(
 
     def get_orders(
         self,
+        workspace_orders: bool = True,
         return_json: bool = False,
         limit: Optional[int] = None,
         sortby: str = "createdAt",
         descending: bool = True,
         order_type: Optional[str] = None,
+        status: Optional[List[str]] = None,
```

Comment:
this is not str, should be literal specifying possible types

Reply:
I took the name from the API_reference, but as I have changed name I will change this as well. :) 👍